tools/cryptogenotyper/cryptogenotyper.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="CryptoGenotyper" name = "CryptoGenotyper" version = "1.0">
+  <description> CryptoGenotyper is a standalone tool to analyze <i>Cryptosporidium</i> Sanger sequencing data and classify the species and subtype based on SSU rRNA and <i>gp60</i> gene markers, respectively. </description>
+  <requirements>
+    <requirement type="package" version ="1.0">cryptogenotyper</requirement>
+  </requirements>
```

Comment:
Could you add in a `<version_command>` section like for [staramr](https://github.com/phac-nml/galaxy_tools/blob/master/tools/staramr/staramr_search.xml#L9). This would be the command to run to print the underlying tool version.

Reply:
I've added it. Thank you